crates/iota-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -151,6 +153,18 @@ impl CertLockGuard {
 
 type JwkAggregator = GenericMultiStakeAggregator<(JwkId, JWK), true>;
 
+/// Represents a scheduling result: a transaction can be either scheduled
+/// for execution, or deferred for some reason. Scheduling result is
+/// returned by the `try_schedule` method of authority per epoch store.
```

Comment:
```suggestion
/// returned by the `try_schedule` method of `AuthorityPerEpochStore`.
```
I would rather use the actual name of the struct, so it's easier to find.